apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -9,7 +9,7 @@ import { EC3Provider } from "@itwin/ec3-widget-react";
 import { SchemaContext } from "@itwin/ecschema-metadata";
 import { ECSchemaRpcLocater } from "@itwin/ecschema-rpcinterface-common";
 import { GeoTools, GeoToolsAddressSearchProvider } from "@itwin/geo-tools-react";
-import { ClientPrefix, GroupingMappingProvider } from "@itwin/grouping-mapping-widget";
+import { CARBON_CALCULATION_BASE_PATH, ClientPrefix, GroupingMappingProvider, REPORTING_BASE_PATH } from "@itwin/grouping-mapping-widget";
```

Comment:
Since you are using the newly imported constants for the OneClickLCA widget, I would expect them to be available from the widget's package, not from unrelated Grouping & Mapping widget package.